src/main/java/com/adobe/target/edge/client/http/DefaultTargetHttpClient.java

```diff
@@ -45,49 +42,46 @@ public class DefaultTargetHttpClient implements TargetHttpClient {
   private static final int DECIMAL_PLACES = 2;
 
   public DefaultTargetHttpClient(ClientConfig clientConfig) {
-    unirestInstance
-        .config()
+    Config config = unirestInstance.config();
+
+    config
         .socketTimeout(clientConfig.getSocketTimeout())
         .connectTimeout(clientConfig.getConnectTimeout())
+        .connectionTTL(Duration.ofMillis(clientConfig.getConnectionTtlMs()))
```

Comment:
could you add a unit test for any of the functionality added here? I believe `DefaultTargetHttpClientTest` would be a good place

Reply:
Added a test